lesson/day01.md

```diff
@@ -5,103 +5,109 @@ title: Day 1
-obtaining which is known as 'ROM dumping'. Please note that obtaining a ROM
-image in a way other than dumping your calculator's ROM to the computer is
-illegal. The tool of choice for a ROM dump is [rom8x](http://www.ticalc.org/archives/files/fileinfo/373/37341.html).
-TiLP can also perform a ROM dump, but rom8x is far more diverse in usability.
-
-To dump your ROM, find the folder named after your calculator. For example, a
-TI-84+CSE would be under 84C, a TI-83+ would be under 83+, and a TI-83+ Silver
-Edition would be under 83S. Send those two .8xp files to your calculator, and
-run them one at a time. You might need to clear out some RAM space, though.
-Send the AppVar created by the program under its name to your computer. Then,
-download the latest OS upgrade file for your model on [TI's website](http://education.ti.com/).
-Then, in the Command Prompt window, navigate to the folder with rom8x.exe,
-the two AppVars, and the OS upgrade file, and type this in:
+**[ED NOTE]: Editor recommendations welcome.**
+
+## Assembler
+
+An assembler translates source code into machine code that the calculator can execute. In
+this guide we will use an assembler called [Brass](http://www.benryves.com/bin/brass/). 
+**[CAPTAIN CALC EDIT]: Should the Doors SDK also be mentioned here?**
+
+The following section will show how to install Brass on Windows, Mac OS, and Linux. **[CAPTAIN CALC EDIT]: The assembler should be placed in the directory for the assembler
+created beforehand**
+
+### Windows
+(Need process description...)
+
+### Mac OS
+(Need process description...)
+
+### Linux
+(Need process description...)
+
+## Emulator
+
+An emulator is a program that runs on a computer and simulates a physical
+calculator. There are several emulators for the TI-83 Plus, such as [WabbitEmu](https://wabbit.codeplex.com/) and [JsTIfied](https://www.cemetech.net/projects/jstified/).
+
+WabbitEmu is an offline emulator that you can download and install like any other computer
+application.
+
+JsTIfied, on the other hand, is an online emulator offered for free by the
+computer/calculator enthusiast site, [Cemetech](https://www.cemetech.net/). To use this 
+emulator you will need to setup a Cemetech account and have an Internet connection. 
+Cemetech also offers a source code editor called [SourceCoder](https://www.cemetech.net/sc/) that allows you to create and save programs on the Cemetech 
+site.
+
+### ROM Dumping
+
+Regardless of the emulator you choose, you will need a ROM image in order to use it. A 
+ROM image, or simply ROM, is a copy of the calculator's software. To obtain a ROM image
+from your TI-83 Plus ("ROM dumping"), you can either use [rom8x](http://www.ticalc.org/archives/files/fileinfo/373/37341.html) or TiLP. Please note that
+you must get the ROM image from a calculator that you own. Obtaining a ROM image from 
+anywhere else is illegal.
+
+**[CAPTAIN CALC EDIT]: Should the following be kept, or should it be replaced with a link
```

Comment:
Best just link to the rom8x documentation.

Reply:
Replaced with link to rom8x on ticalc.org